.github/workflows/run.yml

```
@@ -0,0 +1,71 @@
+name: Run
+on:
+  schedule:
+    - cron: |
+        0 0 * * *
+  pull_request:
+concurrency:
+  group: ${{ github.head_ref || github.sha }}-${{ github.workflow }}
```

We can use literal here because this is not related to a commit. This is only ran periodically.

```suggestion
  group: test
```

---

Oh. I see.